includes/api/save-abandonment-data.php

```diff
@@ -291,6 +298,433 @@ function saveWooCartAbandonmentRecoveryData($razorpayData)
     return $logObj;
 }
 
+//Save cart abandonment data for CartBounty plugin
+function saveCartBountyData($razorpayData){
+    global $wpdb;
+    $products        = WC()->cart->get_cart();
+    $ghost           = false;
+    $name            = $razorpayData['customer_details']['billing_address']['name'] ?? '';
+    $surname         = " ";
+    $email           = $razorpayData['customer_details']['email'];
+    $phone           = $razorpayData['customer_details']['contact'];
+    $cart_table      = $wpdb->prefix ."cartbounty";
```

Comment:
@angbpy maintains variable names in camelcase.

Reply:
Done and updated